src/main/kotlin/com/wgslfuzz/core/Helpers.kt

```diff
@@ -65,3 +65,15 @@ fun isStatementFunctionCallBuiltin(functionCall: Statement.FunctionCall): Boolea
         )
     return functionCall.callee in statementFunctionCallBuiltins
 }
+
```

Comment:
Fine for this PR - but perhaps these should be directly part of the `Type` interface since they're very obviously useful. The only upside I can see to making them extension methods is that default methods on interfaces cannot be `final`, and these are methods one would certainly not want to change the behaviour of. But even if we keep them as extensions, arguably `Types` would be a better location for them. There would be no reduction in cohesion by having them in `Types`.

@Max-Sepp any thoughts on this?